app/pages/lab/subject-set.cjsx

```diff
@@ -17,9 +17,6 @@ VALID_SUBJECT_EXTENSIONS = ['.jpg', '.jpeg', '.png', '.gif', '.svg']
 INVALID_FILENAME_CHARS = ['/', '\\', ':', ',']
 MAX_FILE_SIZE = 600 * 1024
 
-announceSetChange = ->
-  apiClient.type('subject_sets').emit 'add-or-remove'
```

Comment:
Very helpful, thanks @eatyourgreens! In addition to clearing the subject queue on log out, I believe #emptySubjectQueue() is called by #announceSetChange() so that a user editing a project's subjects and subject-sets does not encounter an outdated subject queue when testing a workflow. I'll add #announceSetChange() back into `subject-set.cjsx`.